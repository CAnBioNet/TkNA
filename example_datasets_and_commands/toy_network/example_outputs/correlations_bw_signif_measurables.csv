Edge name,partner1,partner2,Edge Type,Pearson corr pval (Expt 1),Pearson corr pval (Expt 2),Combined corr Pvalue,rho Coefficient (Expt 1),rho Coefficient (Expt 2),Mean rho Coefficient,Combined corr FDR,partner1 name,partner1_MedianValue (Expt 1),partner1_MedianValue (Expt 2),partner1_MeanValue (Expt 1),partner1_MeanValue (Expt 2),partner1_MedianLog2FoldChange (Expt 1),partner1_MedianLog2FoldChange (Expt 2),partner1_MeanLog2FoldChange (Expt 1),partner1_MeanLog2FoldChange (Expt 2),partner2 name,partner2_MedianValue (Expt 1),partner2_MedianValue (Expt 2),partner2_MeanValue (Expt 1),partner2_MeanValue (Expt 2),partner2_MedianLog2FoldChange (Expt 1),partner2_MedianLog2FoldChange (Expt 2),partner2_MeanLog2FoldChange (Expt 1),partner2_MeanLog2FoldChange (Expt 2),Correlations Passed Consistency Filter (All Agree),All Non-PUC Filters Passed,combined Coefficient correlation Direction,partner1_FC_direction,partner2_FC_direction,IfFoldChangeDirectionMatch,PUC,"Final Network Value (0: No edge, 1: Positive edge, -1: Negative edge)"
phenoA<==>phenoB,phenoA,phenoB,pheno<==>pheno,0.07923133740558574,0.07923133740558574,0.03810987476820658,0.5793429920898184,0.5793429920898184,0.5793429920898184,0.05716481215230987,phenoA,0.0405558315,0.0405558315,1.8765651524499998,1.8765651524499998,6.345230542638304,6.345230542638304,7.736600295513288,7.736600295513288,phenoB,0.929664882,0.929664882,1.9220648290999995,1.9220648290999995,3.248974197277172,3.248974197277172,3.2412302817834298,3.2412302817834298,True,True,1.0,1.0,1.0,1.0,1,1.0
phenoA<==>phenoC,phenoA,phenoC,pheno<==>pheno,0.8282202564814,0.8282202564814,0.9445187681694343,-0.0790194400587649,-0.0790194400587649,-0.0790194400587649,0.9445187681694343,phenoA,0.0405558315,0.0405558315,1.8765651524499998,1.8765651524499998,6.345230542638304,6.345230542638304,7.736600295513288,7.736600295513288,phenoC,5.2287738065000005,5.2287738065000005,7.271495752999999,7.271495752999999,-3.321928094887362,-3.321928094887362,-3.321928094898275,-3.321928094898275,True,False,-1.0,1.0,-1.0,-1.0,1,0.0
phenoA<==>phenoD,phenoA,phenoD,pheno<==>pheno,0.29266000532258324,0.29266000532258324,0.296133382487995,-0.36997419157211703,-0.36997419157211703,-0.36997419157211703,0.355360058985594,phenoA,0.0405558315,0.0405558315,1.8765651524499998,1.8765651524499998,6.345230542638304,6.345230542638304,7.736600295513288,7.736600295513288,phenoD,1.5595283235,1.5595283235,5.213366486450001,5.213366486450001,-3.3219280952104357,-3.3219280952104357,-3.321928094750381,-3.321928094750381,True,False,-1.0,1.0,-1.0,-1.0,1,0.0
phenoA<==>geneE,phenoA,geneE,pheno<==>gene,0.08642203736592723,0.08642203736592723,0.04404351591344755,0.5684455204105898,0.5684455204105898,0.5684455204105898,0.34772866492141546,phenoA,0.0405558315,0.0405558315,1.8765651524499998,1.8765651524499998,6.345230542638304,6.345230542638304,7.736600295513288,7.736600295513288,geneE,0.6042385754999999,0.6042385754999999,1.2122117641499999,1.2122117641499999,2.238596208171247,2.238596208171247,1.6796962518841922,1.6796962518841922,True,True,1.0,1.0,1.0,1.0,1,1.0
phenoA<==>geneF,phenoA,geneF,pheno<==>gene,0.6064700605788811,0.6064700605788811,0.7356853682057286,-0.18622429409814367,-0.18622429409814367,-0.18622429409814367,0.7801475059704892,phenoA,0.0405558315,0.0405558315,1.8765651524499998,1.8765651524499998,6.345230542638304,6.345230542638304,7.736600295513288,7.736600295513288,geneF,10.0565699,10.0565699,7.288331885000001,7.288331885000001,-2.62745682738324,-2.62745682738324,-1.1940689076113304,-1.1940689076113304,True,False,-1.0,1.0,-1.0,-1.0,1,0.0
phenoA<==>geneG,phenoA,geneG,pheno<==>gene,0.38389221197291556,0.38389221197291556,0.4295615654828092,-0.3096776086720108,-0.3096776086720108,-0.3096776086720108,0.6443423482242139,phenoA,0.0405558315,0.0405558315,1.8765651524499998,1.8765651524499998,6.345230542638304,6.345230542638304,7.736600295513288,7.736600295513288,geneG,0.23750441900000002,0.23750441900000002,0.9765835356,0.9765835356,2.2882110296139087,2.2882110296139087,3.1973516109963396,3.1973516109963396,True,False,-1.0,1.0,1.0,1.0,0,0.0
phenoB<==>phenoC,phenoB,phenoC,pheno<==>pheno,0.0641703684849123,0.0641703684849123,0.026734752723508733,-0.6044458181469112,-0.6044458181469112,-0.6044458181469112,0.053469505447017465,phenoB,0.929664882,0.929664882,1.9220648290999995,1.9220648290999995,3.248974197277172,3.248974197277172,3.2412302817834298,3.2412302817834298,phenoC,5.2287738065000005,5.2287738065000005,7.271495752999999,7.271495752999999,-3.321928094887362,-3.321928094887362,-3.321928094898275,-3.321928094898275,True,True,-1.0,1.0,-1.0,-1.0,1,-1.0
phenoB<==>phenoD,phenoB,phenoD,pheno<==>pheno,0.011356130222475556,0.011356130222475556,0.0012839419958379935,-0.7563271513870781,-0.7563271513870781,-0.7563271513870781,0.0038518259875139804,phenoB,0.929664882,0.929664882,1.9220648290999995,1.9220648290999995,3.248974197277172,3.248974197277172,3.2412302817834298,3.2412302817834298,phenoD,1.5595283235,1.5595283235,5.213366486450001,5.213366486450001,-3.3219280952104357,-3.3219280952104357,-3.321928094750381,-3.321928094750381,True,True,-1.0,1.0,-1.0,-1.0,1,-1.0
phenoB<==>geneE,phenoB,geneE,pheno<==>gene,0.13100911863835468,0.13100911863835468,0.08693216623035387,0.5111990289088078,0.5111990289088078,0.5111990289088078,0.34772866492141546,phenoB,0.929664882,0.929664882,1.9220648290999995,1.9220648290999995,3.248974197277172,3.248974197277172,3.2412302817834298,3.2412302817834298,geneE,0.6042385754999999,0.6042385754999999,1.2122117641499999,1.2122117641499999,2.238596208171247,2.238596208171247,1.6796962518841922,1.6796962518841922,True,False,1.0,1.0,1.0,1.0,1,0.0
phenoB<==>geneF,phenoB,geneF,pheno<==>gene,0.6443502641101786,0.6443502641101786,0.7801475059704892,-0.16717721388611811,-0.16717721388611811,-0.16717721388611811,0.7801475059704892,phenoB,0.929664882,0.929664882,1.9220648290999995,1.9220648290999995,3.248974197277172,3.248974197277172,3.2412302817834298,3.2412302817834298,geneF,10.0565699,10.0565699,7.288331885000001,7.288331885000001,-2.62745682738324,-2.62745682738324,-1.1940689076113304,-1.1940689076113304,True,False,-1.0,1.0,-1.0,-1.0,1,0.0
phenoB<==>geneG,phenoB,geneG,pheno<==>gene,0.3756218935068804,0.3756218935068804,0.41739816916921535,-0.31482081237558274,-0.31482081237558274,-0.31482081237558274,0.6443423482242139,phenoB,0.929664882,0.929664882,1.9220648290999995,1.9220648290999995,3.248974197277172,3.248974197277172,3.2412302817834298,3.2412302817834298,geneG,0.23750441900000002,0.23750441900000002,0.9765835356,0.9765835356,2.2882110296139087,2.2882110296139087,3.1973516109963396,3.1973516109963396,True,False,-1.0,1.0,1.0,1.0,0,0.0
phenoC<==>phenoD,phenoC,phenoD,pheno<==>pheno,0.0007234970415914797,0.0007234970415914797,8.093986020216944e-06,0.8824370576924025,0.8824370576924025,0.8824370576924025,4.8563916121301665e-05,phenoC,5.2287738065000005,5.2287738065000005,7.271495752999999,7.271495752999999,-3.321928094887362,-3.321928094887362,-3.321928094898275,-3.321928094898275,phenoD,1.5595283235,1.5595283235,5.213366486450001,5.213366486450001,-3.3219280952104357,-3.3219280952104357,-3.321928094750381,-3.321928094750381,True,True,1.0,-1.0,-1.0,1.0,1,1.0
phenoC<==>geneE,phenoC,geneE,pheno<==>gene,0.48167885810419064,0.48167885810419064,0.5709773714482129,-0.25242541017870557,-0.25242541017870557,-0.25242541017870557,0.7613031619309506,phenoC,5.2287738065000005,5.2287738065000005,7.271495752999999,7.271495752999999,-3.321928094887362,-3.321928094887362,-3.321928094898275,-3.321928094898275,geneE,0.6042385754999999,0.6042385754999999,1.2122117641499999,1.2122117641499999,2.238596208171247,2.238596208171247,1.6796962518841922,1.6796962518841922,True,False,-1.0,-1.0,1.0,-1.0,1,0.0
phenoC<==>geneF,phenoC,geneF,pheno<==>gene,0.5494861033371743,0.5494861033371743,0.6635152750402911,0.21570913694660387,0.21570913694660387,0.21570913694660387,0.7801475059704892,phenoC,5.2287738065000005,5.2287738065000005,7.271495752999999,7.271495752999999,-3.321928094887362,-3.321928094887362,-3.321928094898275,-3.321928094898275,geneF,10.0565699,10.0565699,7.288331885000001,7.288331885000001,-2.62745682738324,-2.62745682738324,-1.1940689076113304,-1.1940689076113304,True,False,1.0,-1.0,-1.0,1.0,1,0.0
phenoC<==>geneG,phenoC,geneG,pheno<==>gene,0.23774972760876514,0.23774972760876514,0.21892521698377515,0.4112269751085792,0.4112269751085792,0.4112269751085792,0.4378504339675503,phenoC,5.2287738065000005,5.2287738065000005,7.271495752999999,7.271495752999999,-3.321928094887362,-3.321928094887362,-3.321928094898275,-3.321928094898275,geneG,0.23750441900000002,0.23750441900000002,0.9765835356,0.9765835356,2.2882110296139087,2.2882110296139087,3.1973516109963396,3.1973516109963396,True,False,1.0,-1.0,1.0,-1.0,0,0.0
phenoD<==>geneE,phenoD,geneE,pheno<==>gene,0.11379715876727987,0.11379715876727987,0.06923834213927174,-0.5315779859780297,-0.5315779859780297,-0.5315779859780297,0.34772866492141546,phenoD,1.5595283235,1.5595283235,5.213366486450001,5.213366486450001,-3.3219280952104357,-3.3219280952104357,-3.321928094750381,-3.321928094750381,geneE,0.6042385754999999,0.6042385754999999,1.2122117641499999,1.2122117641499999,2.238596208171247,2.238596208171247,1.6796962518841922,1.6796962518841922,True,False,-1.0,-1.0,1.0,-1.0,1,0.0
phenoD<==>geneF,phenoD,geneF,pheno<==>gene,0.2144659289553039,0.2144659289553039,0.18762579707832436,0.4303253410524978,0.4303253410524978,0.4303253410524978,0.4378504339675503,phenoD,1.5595283235,1.5595283235,5.213366486450001,5.213366486450001,-3.3219280952104357,-3.3219280952104357,-3.321928094750381,-3.321928094750381,geneF,10.0565699,10.0565699,7.288331885000001,7.288331885000001,-2.62745682738324,-2.62745682738324,-1.1940689076113304,-1.1940689076113304,True,False,1.0,-1.0,-1.0,1.0,1,0.0
phenoD<==>geneG,phenoD,geneG,pheno<==>gene,0.23189150274217796,0.23189150274217796,0.2109525632911527,0.41592838339109456,0.41592838339109456,0.41592838339109456,0.4378504339675503,phenoD,1.5595283235,1.5595283235,5.213366486450001,5.213366486450001,-3.3219280952104357,-3.3219280952104357,-3.321928094750381,-3.321928094750381,geneG,0.23750441900000002,0.23750441900000002,0.9765835356,0.9765835356,2.2882110296139087,2.2882110296139087,3.1973516109963396,3.1973516109963396,True,False,1.0,-1.0,1.0,-1.0,0,0.0
geneE<==>geneF,geneE,geneF,gene<==>gene,0.23106424314547172,0.23106424314547172,0.20983174323611303,-0.4165976673276287,-0.4165976673276287,-0.4165976673276287,0.6294952297083392,geneE,0.6042385754999999,0.6042385754999999,1.2122117641499999,1.2122117641499999,2.238596208171247,2.238596208171247,1.6796962518841922,1.6796962518841922,geneF,10.0565699,10.0565699,7.288331885000001,7.288331885000001,-2.62745682738324,-2.62745682738324,-1.1940689076113304,-1.1940689076113304,True,True,-1.0,1.0,-1.0,-1.0,1,-1.0
geneE<==>geneG,geneE,geneG,gene<==>gene,0.9614423092252469,0.9614423092252469,0.9970651988323839,0.0176318533335005,0.0176318533335005,0.0176318533335005,0.9970651988323839,geneE,0.6042385754999999,0.6042385754999999,1.2122117641499999,1.2122117641499999,2.238596208171247,2.238596208171247,1.6796962518841922,1.6796962518841922,geneG,0.23750441900000002,0.23750441900000002,0.9765835356,0.9765835356,2.2882110296139087,2.2882110296139087,3.1973516109963396,3.1973516109963396,True,True,1.0,1.0,1.0,1.0,1,1.0
geneF<==>geneG,geneF,geneG,gene<==>gene,0.7161975981018766,0.7161975981018766,0.8553762293570769,-0.13201503628139225,-0.13201503628139225,-0.13201503628139225,0.9970651988323839,geneF,10.0565699,10.0565699,7.288331885000001,7.288331885000001,-2.62745682738324,-2.62745682738324,-1.1940689076113304,-1.1940689076113304,geneG,0.23750441900000002,0.23750441900000002,0.9765835356,0.9765835356,2.2882110296139087,2.2882110296139087,3.1973516109963396,3.1973516109963396,True,True,-1.0,-1.0,1.0,-1.0,1,-1.0
