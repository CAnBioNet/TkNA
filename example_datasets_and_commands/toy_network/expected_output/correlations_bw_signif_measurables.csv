Edge name,partner1,partner2,Edge Type,Pearson corr pval (Expt 1),Pearson corr pval (Expt 2),Combined corr Pvalue,Corr Coefficient (Expt 1),Corr Coefficient (Expt 2),Mean Corr Coefficient,Combined corr FDR,partner1 name,partner1_MedianValue (Expt 1),partner1_MedianValue (Expt 2),partner1_MeanValue (Expt 1),partner1_MeanValue (Expt 2),partner1_MedianLog2FoldChange (Expt 1),partner1_MedianLog2FoldChange (Expt 2),partner1_MeanLog2FoldChange (Expt 1),partner1_MeanLog2FoldChange (Expt 2),partner2 name,partner2_MedianValue (Expt 1),partner2_MedianValue (Expt 2),partner2_MeanValue (Expt 1),partner2_MeanValue (Expt 2),partner2_MedianLog2FoldChange (Expt 1),partner2_MedianLog2FoldChange (Expt 2),partner2_MeanLog2FoldChange (Expt 1),partner2_MeanLog2FoldChange (Expt 2),Correlations Passed Consistency Filter (All Agree),All Non-PUC Filters Passed,combined Coefficient correlation Direction,partner1_FC_direction,partner2_FC_direction,IfFoldChangeDirectionMatch,PUC,"Final Network Value (0: No edge, 1: Positive edge, -1: Negative edge)"
phenoA<==>phenoB,phenoA,phenoB,pheno<==>pheno,0.079231337405586,0.079231337405586,0.038109874768207,0.579342992089818,0.579342992089818,0.579342992089818,0.05716481215231,phenoA,0.0405558315,0.0405558315,1.87656515245,1.87656515245,6.3452305426383,6.3452305426383,7.73660029551329,7.73660029551329,phenoB,0.929664882,0.929664882,1.9220648291,1.9220648291,3.24897419727717,3.24897419727717,3.24123028178343,3.24123028178343,True,True,1,1,1,1,1,1
phenoA<==>phenoC,phenoA,phenoC,pheno<==>pheno,0.8282202564814,0.8282202564814,0.944518768169434,-0.079019440058765,-0.079019440058765,-0.079019440058765,0.944518768169434,phenoA,0.0405558315,0.0405558315,1.87656515245,1.87656515245,6.3452305426383,6.3452305426383,7.73660029551329,7.73660029551329,phenoC,5.2287738065,5.2287738065,7.271495753,7.271495753,-3.32192809488736,-3.32192809488736,-3.32192809489827,-3.32192809489827,True,False,-1,1,-1,-1,1,0
phenoA<==>phenoD,phenoA,phenoD,pheno<==>pheno,0.292660005322583,0.292660005322583,0.296133382487995,-0.369974191572117,-0.369974191572117,-0.369974191572117,0.355360058985594,phenoA,0.0405558315,0.0405558315,1.87656515245,1.87656515245,6.3452305426383,6.3452305426383,7.73660029551329,7.73660029551329,phenoD,1.5595283235,1.5595283235,5.21336648645,5.21336648645,-3.32192809521044,-3.32192809521044,-3.32192809475038,-3.32192809475038,True,False,-1,1,-1,-1,1,0
phenoA<==>geneE,phenoA,geneE,pheno<==>gene,0.086422037365927,0.086422037365927,0.044043515913448,0.56844552041059,0.56844552041059,0.56844552041059,0.347728664921416,phenoA,0.0405558315,0.0405558315,1.87656515245,1.87656515245,6.3452305426383,6.3452305426383,7.73660029551329,7.73660029551329,geneE,0.6042385755,0.6042385755,1.21221176415,1.21221176415,2.23859620817125,2.23859620817125,1.67969625188419,1.67969625188419,True,True,1,1,1,1,1,1
phenoA<==>geneF,phenoA,geneF,pheno<==>gene,0.606470060578881,0.606470060578881,0.735685368205728,-0.186224294098144,-0.186224294098144,-0.186224294098144,0.780147505970489,phenoA,0.0405558315,0.0405558315,1.87656515245,1.87656515245,6.3452305426383,6.3452305426383,7.73660029551329,7.73660029551329,geneF,10.0565699,10.0565699,7.288331885,7.288331885,-2.62745682738324,-2.62745682738324,-1.19406890761133,-1.19406890761133,True,False,-1,1,-1,-1,1,0
phenoA<==>geneG,phenoA,geneG,pheno<==>gene,0.383892211972916,0.383892211972916,0.429561565482809,-0.309677608672011,-0.309677608672011,-0.309677608672011,0.644342348224214,phenoA,0.0405558315,0.0405558315,1.87656515245,1.87656515245,6.3452305426383,6.3452305426383,7.73660029551329,7.73660029551329,geneG,0.237504419,0.237504419,0.9765835356,0.9765835356,2.28821102961391,2.28821102961391,3.19735161099634,3.19735161099634,True,False,-1,1,1,1,0,0
phenoB<==>phenoC,phenoB,phenoC,pheno<==>pheno,0.064170368484912,0.064170368484912,0.026734752723509,-0.604445818146911,-0.604445818146911,-0.604445818146911,0.053469505447018,phenoB,0.929664882,0.929664882,1.9220648291,1.9220648291,3.24897419727717,3.24897419727717,3.24123028178343,3.24123028178343,phenoC,5.2287738065,5.2287738065,7.271495753,7.271495753,-3.32192809488736,-3.32192809488736,-3.32192809489827,-3.32192809489827,True,True,-1,1,-1,-1,1,-1
phenoB<==>phenoD,phenoB,phenoD,pheno<==>pheno,0.011356130222476,0.011356130222476,0.001283941995838,-0.756327151387078,-0.756327151387078,-0.756327151387078,0.003851825987514,phenoB,0.929664882,0.929664882,1.9220648291,1.9220648291,3.24897419727717,3.24897419727717,3.24123028178343,3.24123028178343,phenoD,1.5595283235,1.5595283235,5.21336648645,5.21336648645,-3.32192809521044,-3.32192809521044,-3.32192809475038,-3.32192809475038,True,True,-1,1,-1,-1,1,-1
phenoB<==>geneE,phenoB,geneE,pheno<==>gene,0.131009118638355,0.131009118638355,0.086932166230354,0.511199028908808,0.511199028908808,0.511199028908808,0.347728664921416,phenoB,0.929664882,0.929664882,1.9220648291,1.9220648291,3.24897419727717,3.24897419727717,3.24123028178343,3.24123028178343,geneE,0.6042385755,0.6042385755,1.21221176415,1.21221176415,2.23859620817125,2.23859620817125,1.67969625188419,1.67969625188419,True,False,1,1,1,1,1,0
phenoB<==>geneF,phenoB,geneF,pheno<==>gene,0.644350264110178,0.644350264110178,0.780147505970489,-0.167177213886118,-0.167177213886118,-0.167177213886118,0.780147505970489,phenoB,0.929664882,0.929664882,1.9220648291,1.9220648291,3.24897419727717,3.24897419727717,3.24123028178343,3.24123028178343,geneF,10.0565699,10.0565699,7.288331885,7.288331885,-2.62745682738324,-2.62745682738324,-1.19406890761133,-1.19406890761133,True,False,-1,1,-1,-1,1,0
phenoB<==>geneG,phenoB,geneG,pheno<==>gene,0.37562189350688,0.37562189350688,0.417398169169215,-0.314820812375583,-0.314820812375583,-0.314820812375583,0.644342348224214,phenoB,0.929664882,0.929664882,1.9220648291,1.9220648291,3.24897419727717,3.24897419727717,3.24123028178343,3.24123028178343,geneG,0.237504419,0.237504419,0.9765835356,0.9765835356,2.28821102961391,2.28821102961391,3.19735161099634,3.19735161099634,True,False,-1,1,1,1,0,0
phenoC<==>phenoD,phenoC,phenoD,pheno<==>pheno,0.000723497041591,0.000723497041591,8.09398602021694E-06,0.882437057692403,0.882437057692403,0.882437057692403,4.85639161213017E-05,phenoC,5.2287738065,5.2287738065,7.271495753,7.271495753,-3.32192809488736,-3.32192809488736,-3.32192809489827,-3.32192809489827,phenoD,1.5595283235,1.5595283235,5.21336648645,5.21336648645,-3.32192809521044,-3.32192809521044,-3.32192809475038,-3.32192809475038,True,True,1,-1,-1,1,1,1
phenoC<==>geneE,phenoC,geneE,pheno<==>gene,0.481678858104191,0.481678858104191,0.570977371448213,-0.252425410178706,-0.252425410178706,-0.252425410178706,0.76130316193095,phenoC,5.2287738065,5.2287738065,7.271495753,7.271495753,-3.32192809488736,-3.32192809488736,-3.32192809489827,-3.32192809489827,geneE,0.6042385755,0.6042385755,1.21221176415,1.21221176415,2.23859620817125,2.23859620817125,1.67969625188419,1.67969625188419,True,False,-1,-1,1,-1,1,0
phenoC<==>geneF,phenoC,geneF,pheno<==>gene,0.549486103337175,0.549486103337175,0.663515275040291,0.215709136946604,0.215709136946604,0.215709136946604,0.780147505970489,phenoC,5.2287738065,5.2287738065,7.271495753,7.271495753,-3.32192809488736,-3.32192809488736,-3.32192809489827,-3.32192809489827,geneF,10.0565699,10.0565699,7.288331885,7.288331885,-2.62745682738324,-2.62745682738324,-1.19406890761133,-1.19406890761133,True,False,1,-1,-1,1,1,0
phenoC<==>geneG,phenoC,geneG,pheno<==>gene,0.237749727608765,0.237749727608765,0.218925216983775,0.411226975108579,0.411226975108579,0.411226975108579,0.43785043396755,phenoC,5.2287738065,5.2287738065,7.271495753,7.271495753,-3.32192809488736,-3.32192809488736,-3.32192809489827,-3.32192809489827,geneG,0.237504419,0.237504419,0.9765835356,0.9765835356,2.28821102961391,2.28821102961391,3.19735161099634,3.19735161099634,True,False,1,-1,1,-1,0,0
phenoD<==>geneE,phenoD,geneE,pheno<==>gene,0.11379715876728,0.11379715876728,0.069238342139272,-0.53157798597803,-0.53157798597803,-0.53157798597803,0.347728664921416,phenoD,1.5595283235,1.5595283235,5.21336648645,5.21336648645,-3.32192809521044,-3.32192809521044,-3.32192809475038,-3.32192809475038,geneE,0.6042385755,0.6042385755,1.21221176415,1.21221176415,2.23859620817125,2.23859620817125,1.67969625188419,1.67969625188419,True,False,-1,-1,1,-1,1,0
phenoD<==>geneF,phenoD,geneF,pheno<==>gene,0.214465928955304,0.214465928955304,0.187625797078324,0.430325341052498,0.430325341052498,0.430325341052498,0.43785043396755,phenoD,1.5595283235,1.5595283235,5.21336648645,5.21336648645,-3.32192809521044,-3.32192809521044,-3.32192809475038,-3.32192809475038,geneF,10.0565699,10.0565699,7.288331885,7.288331885,-2.62745682738324,-2.62745682738324,-1.19406890761133,-1.19406890761133,True,False,1,-1,-1,1,1,0
phenoD<==>geneG,phenoD,geneG,pheno<==>gene,0.231891502742178,0.231891502742178,0.210952563291153,0.415928383391095,0.415928383391095,0.415928383391095,0.43785043396755,phenoD,1.5595283235,1.5595283235,5.21336648645,5.21336648645,-3.32192809521044,-3.32192809521044,-3.32192809475038,-3.32192809475038,geneG,0.237504419,0.237504419,0.9765835356,0.9765835356,2.28821102961391,2.28821102961391,3.19735161099634,3.19735161099634,True,False,1,-1,1,-1,0,0
geneE<==>geneF,geneE,geneF,gene<==>gene,0.231064243145472,0.231064243145472,0.209831743236113,-0.416597667327629,-0.416597667327629,-0.416597667327629,0.62949522970834,geneE,0.6042385755,0.6042385755,1.21221176415,1.21221176415,2.23859620817125,2.23859620817125,1.67969625188419,1.67969625188419,geneF,10.0565699,10.0565699,7.288331885,7.288331885,-2.62745682738324,-2.62745682738324,-1.19406890761133,-1.19406890761133,True,True,-1,1,-1,-1,1,-1
geneE<==>geneG,geneE,geneG,gene<==>gene,0.961442309225246,0.961442309225246,0.997065198832384,0.017631853333501,0.017631853333501,0.017631853333501,0.997065198832384,geneE,0.6042385755,0.6042385755,1.21221176415,1.21221176415,2.23859620817125,2.23859620817125,1.67969625188419,1.67969625188419,geneG,0.237504419,0.237504419,0.9765835356,0.9765835356,2.28821102961391,2.28821102961391,3.19735161099634,3.19735161099634,True,True,1,1,1,1,1,1
geneF<==>geneG,geneF,geneG,gene<==>gene,0.716197598101876,0.716197598101876,0.855376229357077,-0.132015036281392,-0.132015036281392,-0.132015036281392,0.997065198832384,geneF,10.0565699,10.0565699,7.288331885,7.288331885,-2.62745682738324,-2.62745682738324,-1.19406890761133,-1.19406890761133,geneG,0.237504419,0.237504419,0.9765835356,0.9765835356,2.28821102961391,2.28821102961391,3.19735161099634,3.19735161099634,True,True,-1,-1,1,-1,1,-1
